frontend/site-registry/src/app/features/details/SiteDetails.tsx

```diff
@@ -155,36 +158,29 @@ const SiteDetails = () => {
 
   const saveSiteDetailsRequestStatus = useSelector(saveRequestStatus);
 
-  useEffect(()=>{
-
-    if(saveSiteDetailsRequestStatus === RequestStatus.success || saveSiteDetailsRequestStatus === RequestStatus.failed)
-    {
-      if(saveSiteDetailsRequestStatus === RequestStatus.success)
-      {
+  useEffect(() => {
+    if (
+      saveSiteDetailsRequestStatus === RequestStatus.success ||
+      saveSiteDetailsRequestStatus === RequestStatus.failed
+    ) {
+      if (saveSiteDetailsRequestStatus === RequestStatus.success) {
         dispatch(resetSaveSiteDetails(null));
         dispatch(updateSiteDetailsMode(SiteDetailsMode.ViewOnlyMode));
         setEdit(false);
-      }
-      else
-      {
+      } else {
         // dont close edit mode
       }
 
       showNotification(
         saveSiteDetailsRequestStatus,
         'Successfully saved site details',
-        'Failed To save site details'
+        'Failed To save site details',
```

Comment:
The showNotification function is always called with both success and failure messages, even if the status is not success. This could be optimized to display only relevant notifications based on the status.

Reply:
Same as above, not a part of this functionality.